Update streamlabswater.markdown

fortepc opened the pull request:
I didn't know about the API costs when I purchased the device, so this warning may help others make a more informed decision if they want to buy this product based on it's HA integration like I did.

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase:
- This PR fixes or closes issue:

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

fortepc committed:
Update streamlabswater.markdown

I didn't know about the API costs when I purchased the device, so this warning may help others make a more informed decision if they want to buy this product based on it's HA integration like I did.

frenck reviewed (changes requested):
Let's make it an actual message box, instead of adding just a warning to the introduction text.

fortepc committed:
Moved warning from description to a note box

Wasn't sure if a note box, or a warning box was appropriate, so just used a normal note box. I also added a link to their subscription page instead of specifically listing the prices. That way if the prices change the docs don't need updated.

fortepc committed:
Update streamlabswater.markdown

Fixed broken URL from previous edit

fortepc commented:
Okay, I think I've addressed that needed change.

frenck committed:
:pencil2: Tweak

frenck commented:
Thanks!
